### PR TITLE
WL-4220 LSNBLDR-633: 2 bug fixes

### PR DESCRIPTION
**Pull request by nicholaswilson100**
This is just a copy of the code in the latest PR for LSNBLDR-633 (https://github.com/sakaiproject/sakai/pull/4347) except that I had to move one line to a different place because the method had been moved.

### CODE REVIEW

File: lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java [hunk 1]
```diff
@@ -843,7 +843,11 @@ public SimplePage getPage(long pageId) {
 	}
 
 	public List<SimplePage> getSitePages(String siteId) {
-	    DetachedCriteria d = DetachedCriteria.forClass(SimplePage.class).add(Restrictions.eq("siteId", siteId)).add(Restrictions.isNull("owner"));
+	    DetachedCriteria d = DetachedCriteria.forClass(SimplePage.class).add(Restrictions.eq("siteId", siteId))
+			        .add(Restrictions.disjunction()
```
Comment (buckett) [inline lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java]: Indentation is all over the place here.

File: lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java [hunk 2]
```diff
@@ -843,7 +843,11 @@ public SimplePage getPage(long pageId) {
...
+						.add(Restrictions.isNull("owner"))
+					    .add(Restrictions.conjunction().add(Restrictions.isNotNull("owner")).add(Restrictions.eq("owned", true)))
```
Comment (buckett) [inline lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java]: Can't this just be: `.add(Restrictions.eq("owned", true))` as you test for owner being `null` on the previous option.

**Review by buckett — APPROVED** [review]

**Comment by buckett** [comment]
Could you update the Sakai PR with the changes, thanks.

**Comment by nicholaswilson100** [comment]
yup will do

**Pull request merged**